### DISCUSSION TITLE
Don't automatically use amsmath in PDF output?

### DISCUSSION THREAD

**Discussion by mikemahoney218** [quarto-dev/quarto-cli#1664] [Q&A] [answered]
Hi all!

I'm attempting to port the template for IOP Publications for a Quarto template. Their .cls has unavoidable and non-reconcillable conflicts with `amsmath`:

https://publishingsupport.iopscience.iop.org/questions/latex-template/

I _believe_ that Quarto is automatically including `\usepackage{amsmath}` in the generated output, in order to support rendering math and theorems. Is there a way to disable this behavior so the package is not loaded?

**Comment by cscheid** [comment, marked as answer]
Quarto's usage comes from [pandoc's Latex template](https://github.com/jgm/pandoc/blob/master/data/templates/default.latex).
In your format, you can replace the standard pandoc template with your own completely custom one. You can always start by copying that template as your own, and then making changes to it.

**Reply by mikemahoney218** [reply]
Thanks!

For anyone else looking, docs on replacing templates here:
https://quarto.org/docs/journals/templates.html
